src/applications/caregivers/config/chapters/veteran/vetMedicalCenter_api.js

```diff
@@ -0,0 +1,41 @@
+import fullSchema from 'vets-json-schema/dist/10-10CG-schema.json';
+import constants from 'vets-json-schema/dist/constants.json';
+import { veteranFields } from 'applications/caregivers/definitions/constants';
+import {
+  LastTreatmentFacilityAPIUI,
+  PreferredFacilityAPIUI,
+} from 'applications/caregivers/definitions/UIDefinitions/veteranUI';
+
+const {
+  lastTreatmentFacility,
+  plannedClinic,
+} = fullSchema.properties.veteran.properties;
+const caregiverStates = constants.states50AndDC
+  .concat([{ label: 'Puerto Rico', value: 'PR' }])
```

Comment:
Seems to me that if we include Puerto Rico, we should also include Guam, U.S. Virgin islands, etc...?

Reply:
There are no caregiver facilities on those islands... only PR. Ideally, Id love for this list to be completely generated dynamically with the API, but it doesn't allow you to just search by service. You have to search by service and location.